src/main/java/com/walking/intensive/chapter2/task7/Task7.java

```diff
@@ -28,10 +28,55 @@
 public class Task7 {
     public static void main(String[] args) {
 //        Для собственных проверок можете делать любые изменения в этом методе
+
+        System.out.println(getFriendlyPair(125000));
     }
 
     static int getFriendlyPair(int n) {
-        // Ваш код
-        return 0;
+        if (n < 1 || n > 1000000) {
+            return -1;
+        }
+
+        final int MIN_VALUE = 284;
+
+        int num1, num2;
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter2/task7/Task7.java:42:9: Each variable declaration must be in its own statement. [MultipleVariableDeclarations]